pyproject.toml

```diff
@@ -43,6 +42,10 @@ dependencies = [
 Repository = "https://github.com/FAIRmat-NFDI/nomad-simulation-parsers"
 
 [project.optional-dependencies]
+md = [
+    "nomad-simulations[md]",
```

Comment:
not sure how this works but will this target the release version? or somehow resolved same as in deps